src/express-middleware.js

```diff
@@ -37,12 +41,21 @@ export function createExpressMiddleware(adapter, middlewareOptions = {}) {
       }
 
       // Lastly, send the response
-      res.send(responseOptions.content || '');
+      const content = responseOptions.content || '';
+      debug('response body: %s', content);
+      res.send(content);
+      res.on('finish', () => {
+        // NOTE: uses undocumented API of http.ServerResponse, but OK for logging code
+        // eslint-disable-next-line no-underscore-dangle
```

Comment:
i'm basically justifying my turning off of a linter rule on the next line. the `res._headers` property isn't a documented one from the standard library in node, but its been there for all of eternity. in fact, express depends on it (which like 90% of all web servers written in node use). i just didn't want to leave the pragma for disabling the linter rule and not tell anyone why. i can try to reword to make that clearer.